bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -0,0 +1,420 @@
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.eclipse.smarthome.core.items.GenericItem;
+import org.eclipse.smarthome.core.items.GroupItem;
+import org.eclipse.smarthome.core.items.Item;
+import org.eclipse.smarthome.core.items.ItemNotFoundException;
+import org.eclipse.smarthome.core.items.ItemRegistry;
+import org.eclipse.smarthome.core.items.ItemRegistryChangeListener;
+import org.eclipse.smarthome.core.items.StateChangeListener;
+import org.eclipse.smarthome.core.persistence.FilterCriteria;
+import org.eclipse.smarthome.core.persistence.HistoricItem;
+import org.eclipse.smarthome.core.persistence.PersistenceManager;
+import org.eclipse.smarthome.core.persistence.PersistenceService;
+import org.eclipse.smarthome.core.persistence.PersistenceServiceConfiguration;
+import org.eclipse.smarthome.core.persistence.QueryablePersistenceService;
+import org.eclipse.smarthome.core.persistence.SimpleItemConfiguration;
+import org.eclipse.smarthome.core.persistence.config.SimpleAllConfig;
+import org.eclipse.smarthome.core.persistence.config.SimpleConfig;
+import org.eclipse.smarthome.core.persistence.config.SimpleGroupConfig;
+import org.eclipse.smarthome.core.persistence.config.SimpleItemConfig;
+import org.eclipse.smarthome.core.persistence.strategy.SimpleCronStrategy;
+import org.eclipse.smarthome.core.persistence.strategy.SimpleStrategy;
+import org.eclipse.smarthome.core.scheduler.CronExpression;
+import org.eclipse.smarthome.core.scheduler.ExpressionThreadPoolManager;
+import org.eclipse.smarthome.core.scheduler.ExpressionThreadPoolManager.ExpressionThreadPoolExecutor;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class implements a persistence manager to manage all persistence services etc.
+ *
+ * @author Kai Kreuzer - Initial contribution and API
+ * @author Markus Rathgeb - Separation of persistence core and model, drop Quartz usage.
+ */
+public class PersistenceManagerImpl implements PersistenceManager, ItemRegistryChangeListener, StateChangeListener {
+
+    private final Logger logger = LoggerFactory.getLogger(PersistenceManager.class);
+
+    // the scheduler used for timer events
+    private final ExpressionThreadPoolExecutor scheduler;
+
+    private ItemRegistry itemRegistry;
+
+    final Map<String, PersistenceService> persistenceServices = new HashMap<>();
+    final Map<String, PersistenceServiceConfiguration> persistenceServiceConfigs = new HashMap<>();
+    private final Map<String, Set<Runnable>> persistenceJobs = new HashMap<>();
+
+    public PersistenceManagerImpl() {
+        scheduler = ExpressionThreadPoolManager.getExpressionScheduledPool("persist");
```

Comment:
wouldn't it be better to do this in activate() and de-reference and dispose it again in deactivate()?